ddtrace/contrib/consul/patch.py

```diff
@@ -43,9 +42,10 @@ def trace_func(wrapped, instance, args, kwargs):
             return wrapped(*args, **kwargs)
 
         path = kwargs.get('key') or args[0]
+        resource = '%s %s' % (name.upper(), path)
```

Comment:
now that I think about it... what does the cardinality look like on `path`? do we need to have as just `GET`/`PUT`/etc?